GitUp/Application/AppDelegate.m

```diff
@@ -550,17 +553,9 @@ - (IBAction)viewIssues:(id)sender {
 }
 
 - (IBAction)showAboutPanel:(id)sender {
-#if DEBUG
-  _versionTextField.stringValue = @"DEBUG";
-#else
-  if (_updatePending) {
-    _versionTextField.stringValue = NSLocalizedString(@"Update Pending", nil);
-  } else {
-    _versionTextField.stringValue = [NSString stringWithFormat:NSLocalizedString(@"Version %@ (%@)", nil), [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleShortVersionString"], [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleVersion"]];
-  }
-#endif
-  _copyrightTextField.stringValue = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSHumanReadableCopyright"];
-  [_aboutPanel makeKeyAndOrderFront:nil];
+  self.aboutPanel = [AboutPanel new];
```

Comment:
This will always create a new `AboutPanel` which I guess is ok. I'd rather just lazy load on property access but I won't hold the review back. I'm not sure the property is necessary though if we load it every time. Also, let's stick to `alloc init` paradigm for consistency.